src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,33 @@
 
 class Animal {
   // write your code here
+  static alive = [];
```

Comment:
```suggestion
  Animal.alive = [];
```
let's change it like this and move outside Animal class :)